src/tests/recipe-descriptions-test.ts

```diff
@@ -51,8 +51,8 @@ particle DisplayBox in 'test.js'
   consume root
   description \`ignore this description too\`
 recipe
-  ${options.includeAllStore ? `use 'allboxes'` : `create`} as handle0
-  ${options.includeStore ? `use 'mybox'` : `create`} as handle1
+  ${options.includeAllStore ? `copy 'allboxes'` : `create`} as handle0
+  ${options.includeStore ? `copy 'mybox'` : `create`} as handle1
```

Comment:
why has this test changed?

Reply:
The recipe has changed because it was wrong - it was trying to `use` a store that is in the context, not in the arc. It used to (wrongly) work because while loading the context, the volatile storage would be created and be later accessible via the volatile cache.
I added a test to arc-tests to test the combinations of copy vs use from context and arc.

Reply:
Got it, thanks.